# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/promise/moz.build
# Commit: 0bd696ed34ea
# Full Hash: 0bd696ed34eace2844a540509a5022dc57b48508
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2019-03-27 04:44:00
# Regressor Bug: 1483036
# File Overlap Count: 2
# Description:
#   Bug 1483036 - Add MaybeResolveWithClone and MaybeRejectWithClone to Promise. r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D24385
# ==============================================================================

diff -r b7663bdb3f6e -r 0bd696ed34ea dom/promise/moz.build
--- a/dom/promise/moz.build	Tue Mar 26 19:28:15 2019 +0000
+++ b/dom/promise/moz.build	Tue Mar 26 19:34:29 2019 +0000
@@ -23,6 +23,7 @@
 LOCAL_INCLUDES += [
     '../base',
     '../ipc',
+    '/js/xpconnect/src',
 ]
 
 include('/ipc/chromium/chromium-config.mozbuild')
